utils/checks/features-snapshot

```diff
@@ -4,7 +4,7 @@ cd "$(dirname "${BASH_SOURCE[0]}")/../.."
 
 list() {
   cargo metadata --format-version 1 |
-    jq '.resolve.nodes | map(. | { name: (.id | sub("\\s+(?<version>[^\\s]+?)\\s.*$"; " \(.version)")), features: (.features | sort) }) | sort_by(.name)' |
+    jq '.resolve.nodes | map(. | { name: (.id | if startswith("path+file") then sub(".*/(?<version>)"; "\(.version)") | (split("#")|join(" ")) else sub(".*#(?<version>)"; "\(.version)") | (split("@")|join(" "))  end), features: (.features | sort) }) | sort_by(.name)' |
```

Comment:
Ultimately parsing the Package ID is not a good idea, and we should prefer doing a lookup in the list of packages by the id instead, and using the version from there.

I'll take a look in a bit and see if I can rewrite this with a new form. This is definitely possible with a bit more advanced jq magic since there are variables and we can do cross-subfield lookups with them...

Reply:
Rewrited by myself, looks working.

https://github.com/humanode-network/humanode/pull/1070/commits/fddc1f8530d8c80dfb3430661e2352a93b51a2f4